src/app/containers/manuscript/article-information/index.tsx

```diff
@@ -51,6 +52,9 @@ export const ArticleInformation: React.FC<{}> = () => {
         <div>
           <strong> Published: </strong> {publicationDate.isValid() ? publicationDate.format('MMMM D, YYYY') : ''}
         </div>
+        <div>&nbsp;</div>
+        <div>{articleInfo.copyrightStatement}</div>
+        <div dangerouslySetInnerHTML={{ __html: stringifyEditorState(articleInfo.licenseText) }} />
```

Comment:
Probably best not to use dangerouslySetInnerHTML and use something like https://www.npmjs.com/package/interweave - but you are setting content that you control so not the end of the world